metals/src/main/scala/scala/meta/internal/metals/FindNonSourceTextRequest.scala

```diff
@@ -0,0 +1,6 @@
+package scala.meta.internal.metals
+
+case class FindNonSourceTextRequest(
+    mask: java.lang.String,
```

Comment:
Do we need a mask really? Jars will contain `.conf` files and `.properties` most likely, so not much possible values there.

Reply:
I think there are more plain-text extensions that might be presented in jars and we will never cover them all.
Also the mask might be used not only for filtering by extension but also just by full name like: `sample.json`.

Reply:
I did it by advice from @dos65 and I guess he has a point - some people put xml/json/txt files in jars, so who knows what a user may need to search for